### ISSUE TITLE
Not working - PHP 7

### ISSUE THREAD

**Issue by r3sist** [ptraceur/f3-falsum#8]
Hi!

I would like to use this awsome error handler in my new F3 (3.5.0) app. On my test server the PHP version is 7.0.5, I use F3 and Falsum with Composer. Anyway, Falsum is not working: I get empty page on every error. (eg. on 404) Is it possible it is because of the PHP version? 

If I comment out
`$rows = file($errors[$key]['file']);` 
in [Run.php](https://github.com/ptraceur/f3-falsum/blob/master/src/Falsum/Run.php#L101), I get your dashboard and everything is ok except the file contents.

Do you have any suggestions? Thank you!

**Comment by ptraceur** [comment]
Hey @r3sist,
check on your composer the Falsum version. This was a bug on Unix systems, but I got it corrected earlier this morning. v2.4 should be fine. (Also you may have to change the vendor name from rooftoprafa to ptraceur since I change it today as well)

Let me know if that worked out for you

**Comment by r3sist** [comment]
I will investigate this, thanks

**Comment by ptraceur** [comment]
Tell me something when you do :wink:

**Comment by r3sist** [comment]
So I have a legacy error handler f3 'plugin' (https://github.com/rafamds/f3-error-handler ;) ) The basic difference beetwen that and this version of Falsum code is the source file selection at the sidebar. Right?

And if I insert an echo in your code: `echo $errors[$key]['file'];` I get the one file (index.php) without path:
```
/home/admin/web/.../vendor/bcosca/fatfree/lib/base.php
/home/admin/web/.../web.php
index.php
```
That's why file() in `$rows = file($add_root.$errors[$key]['file']);` can't run. If I hack the $matches[0] to coontain one source file with correct path, everything is ok.

```
app/
   Controller.php
public/   <- document root for web server
   assets/
   index.php  <- problem with this file
ui/
   header.html
vendor/
   ptraceur/
   bcosca/
web.php  <- bootstrapping f3
```

**Comment by ptraceur** [comment]
Yes the only difference is the sidebar with a possibility to view all code. 
I didn't had any time to fix this problem, and I never experienced it while coding.

I will try to replicate this on my computer anytime soon
